src/coreclr/jit/emitarm64.cpp

```diff
@@ -14758,21 +14779,25 @@ void emitter::emitIns_Call(EmitCallType          callType,
         case IF_SVE_AJ_3A:
         case IF_SVE_AL_3A:
         case IF_SVE_CL_3A:
-        case IF_SVE_DD_2A:
         case IF_SVE_DF_2A:
         case IF_SVE_GS_3A:
         case IF_SVE_HJ_3A:
         case IF_SVE_IY_4A:
             return PREDICATE_NONE;
 
+        case IF_SVE_DD_2A:
+            assert((regpos >= 1) && (regpos <= 3));
+            return ((regpos == 2) ? PREDICATE_NONE : PREDICATE_SIZED);
+
         case IF_SVE_CX_4A:
         case IF_SVE_CX_4A_A:
         case IF_SVE_CY_3A:
         case IF_SVE_CY_3B:
+        case IF_SVE_DG_2A:
```

Comment:
From https://docsmirror.github.io/A64/2023-06/rdffr_p_p_f.html, this should be:

```c++
case IF_SVE_DG_2A:
   return ((regpos == 2) ? PREDICATE_ZERO: PREDICATE_NONE);
```

Reply:
Sure, it would avoid asserts and make `IF_SVE_DG_2A` a separate case for groups that have only two registers and those are predicate registers. The other groups also have only two predicate registers but they have non-predicated registers  as well.

Reply:
oh, and we need `PREDICATE_SIZED` for reg1 otherwise we won't get the element type in `<Pd>.B`.